src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Machine {
+    public abstract void doWork() {
+    }
+    public abstract void stopWork() {
+    }
```

Comment:
The class 'Machine' should be declared as abstract because it contains abstract methods 'doWork()' and 'stopWork()'. Without the abstract keyword, the class cannot be instantiated but also cannot have abstract methods. Please add the abstract keyword to the class declaration.